std-bits/table/src/main/java/org/enso/table/data/column/storage/numeric/BigDecimalStorage.java

```diff
@@ -0,0 +1,64 @@
+package org.enso.table.data.column.storage.numeric;
+
+import java.math.BigDecimal;
+import org.enso.table.data.column.operation.map.MapOperationStorage;
+import org.enso.table.data.column.operation.map.numeric.arithmetic.AddOp;
+import org.enso.table.data.column.operation.map.numeric.arithmetic.BigDecimalDivideOp;
+import org.enso.table.data.column.operation.map.numeric.arithmetic.ModOp;
+import org.enso.table.data.column.operation.map.numeric.arithmetic.MulOp;
+import org.enso.table.data.column.operation.map.numeric.arithmetic.PowerOp;
```

Comment:
As with large (BigInteger) integers, we convert to double for this operation. Preserving decimal accuracy for exponentiation in general seems unnecessary.